src/controllers/user.js

```diff
@@ -102,16 +102,22 @@ export const getUserById = async (req, res) => {
 export const updateUserById = async (req, res) => {
     try {
       const id = req.params.id;
-      const [user] = await model.User.update(req.body, {where: {id : id }});
+      const [user] = await model.User.update(
+        {
+          firstname: req.body.firstname,
+          lastname: req.body.lastname,
+          password: req.body.password
```

Comment:
This current implementation is causing a terrible bug coz can change password and store it as plain text
<img width="1440" alt="Screen Shot 2021-02-19 at 18 34 25" src="https://user-images.githubusercontent.com/54619678/108533501-b84a0980-72e1-11eb-9ee7-8b1d6afe9917.png">

Reply:
Thx @SimonImanigirimpuhwe I have added the encryption before we update the password